mteb/tasks/PairClassification/multilingual/XStance.py

```diff
@@ -100,8 +100,8 @@ def dataset_transform(self):
             for split in self.metadata.eval_splits:
                 _dataset[lang][split] = [
                     {
-                        "sent1": self.dataset[lang][split]["sent1"],
-                        "sent2": self.dataset[lang][split]["sent2"],
+                        "sent1": self.dataset[lang][split]["sentence1"],
```

Comment:
I think that key should be "sentence1", because STSEvalutator expecting "sentence1" https://github.com/embeddings-benchmark/mteb/blob/e949d2ac95079de988b00de6c1138e34ac8a4c80/mteb/abstasks/AbsTaskSTS.py#L82-L83

Reply:
It is updated to "sentence1" in this PR. Is there anything that should be changed?

Reply:
I mean like that, but I'm not sure, because it seems to working now 
```suggestion
                        "sentence 1": self.dataset[lang][split]["sentence1"],
```

Reply:
Oh yes, good catch! I'll update that. Right now it fails running
```
mteb run -t XStance -m sentence-transformers/all-MiniLM-L6-v2
```